test/functional/client_side_encryption_functional_tests.js

```diff
@@ -0,0 +1,160 @@
+'use strict';
```

Comment:
nit: calling these "functional tests" inside the `functional` folder is redundant. Should we make a "client_side_encryption" folder, and then just have `corpus_tests.js` and `something_more_descriptive_tests.js`?

Reply:
that looks great to me!